source/util/CommandLine.cpp

```diff
@@ -801,4 +865,32 @@ std::string CommandLine::Option::set(OptionCallback& target, string_view, string
     return target(value);
 }
 
+std::string CommandLine::Option::set(std::map<std::string, int>& target, string_view name,
+                                     string_view value) {
+    size_t equalsIndex = value.find_first_of(',');
+    string_view num_args;
+    if (equalsIndex != string_view::npos) {
+        num_args = value.substr(equalsIndex + 1);
+        value = value.substr(0, equalsIndex);
+        std::string error;
+        auto result = parseInt<uint64_t>(name, num_args, error);
+        if (result)
+            target[std::string(value)] = *result;
+        return error;
+    }
+    return {};
```

Comment:
If there is no comma should that be considered an error? Seems like it to me, because the option on its own is otherwise useless.

Reply:
It is considered an error, but it is handled silently by not accepting the option.
I should have added an explicit error message.